pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.poetry]
+name = "securedrop-workstation-keyring"
+version = "0.0.0"
+description = ""
+authors = ["SecureDrop Team"]
+license = "AGPLv3+"
+
+[tool.poetry.dependencies]
```

Comment:
:upside_down_face: Good point, given if we do any kind of graphical install assistance it will be in the workstation config app.  poetry stuff is gone and the license and author are contained in the spec file, so I have removed pyproject.toml entirely; hopefully not rash.